Common/Common.css

```diff
@@ -0,0 +1,96 @@
+
+:root{
+    --background-color : #E5E5E5;
+    --empty-space-color : #C4C4C4;
+    --color-white : white;
+}
+
+html{
+    font-size: 62.5%;
+    box-sizing: border-box;
+}
```

Comment:
Suggestion: You can move `font-size` to root and remove `box-sizing` from here as it is already mentioned in the `body`.